go/sts/README.md

```diff
@@ -0,0 +1,64 @@
+# AWS SDK for Go code examples for AWS STS
+
+## Purpose
+
+These examples demonstrate how to perform several AWS System Token Service (AWS STS) operations.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the _AWS SDK for Go Developer Guide_.
+
+## Running the code
+
+### TakeRole/TakeRole.go
+
+This example gets temporary security credentials to access resources.
+
+`go run TakeRole.go -r ROLE-ARN -S SESSION-NAME`
+
+- _ROLE-ARN_ is the ARN of the role to assume.
```

Comment:
...the **Amazon Resource Name (ARN)**...

Reply:
Fixed